recipes/openssh/all/conanfile.py

```diff
@@ -0,0 +1,132 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import patch, copy, get, rmdir, export_conandata_patches
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps
+from conan.tools.layout import basic_layout
+from os.path import join
+
+required_conan_version = ">=1.54.0"
+
+
+class PackageConan(ConanFile):
+    name = "openssh"
+    description = "The OpenSSH (portable) suite of secure connectivity tools"
+    license = "BSD"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/openssh/openssh-portable"
+    topics = ("security", "cryptography", "login", "keychain", "file-sharing")
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_openssl": [True, False],
+        "with_pam": [True, False],
```

Comment:
While we don't have a Linux PAM package now, it might be good build this option out in a way where that's possible in the future, i.e.

```suggestion
        "with_pam": [None, "OpenPAM"],
```

```suggestion
        "with_pam": [True, False],
```

Reply:
Updating option